src/pages/home/report/ReportActionsView.js

```diff
@@ -189,9 +188,6 @@ class ReportActionsView extends React.Component {
         unsubscribeFromReportChannel(oldReportID);
         subscribeToReportTypingEvents(this.props.reportID);
         Timing.end(CONST.TIMING.SWITCH_REPORT, CONST.TIMING.COLD);
-
-        // Fetch the new set of actions
-        fetchActions(this.props.reportID);
```

Comment:
Removed fetch calls here. As they are performed in the ReportScreen when reportID changes

There might be other items that can be cleaned up now that the component re-mounts on report change